share/templates.d/99-generic/runtime-cleanup.tmpl

```diff
@@ -240,10 +240,37 @@ removefrom libvorbis /usr/${libdir}/libvorbisenc.*
 removefrom libxml2 /usr/bin/*
 removefrom libxml2-python /usr/${libdir}/python?.?/site-packages/libxml2mod.a
 removefrom libxml2-python /usr/${libdir}/python?.?/site-packages/libxml2mod.la
+removefrom linux-firmware /usr/lib/firmware/dvb*
+removefrom linux-firmware /usr/lib/firmware/*_12mhz*
+removefrom linux-firmware /usr/lib/firmware/v4l*
+removefrom linux-firmware /usr/lib/firmware/brcm/BCM-*
```

Comment:
This looks like it may include some broadcom related wifi firmware, and some things related to Raspberry Pi.
I don't know how to tell for sure if they are important, do you?

Reply:
The first three are all TV capture stuff. The fourth I meant to capture the very first driver in the `WHENCE` list, which is documented as "Cypress Bluetooth firmware for HTC Vive" - the Vive is a VR headset. I did stop removing *anything* to do with bluetooth, wifi, serial ports, PCMCIA etc. after that though, and I guess the wildcard might inadvertently capture a similarly-named thing added later. We could drop that line I guess.

Reply:
Ah, right, looks like they named everything else with the brcm prefix. It can stay.